src/python/pants/option/parser.py

```diff
@@ -565,7 +565,7 @@ def expand(val_or_str):
                     fromfile = val_or_str[1:]
                     try:
                         with open(fromfile) as fp:
-                            s = fp.read().strip()
+                            s = fp.read()
```

Comment:
This should never have done stripping, and I was probably the one that added it... 🤦

Reply:
Well some tests are failing without it, lol